lib/dialog-renderer/index.tsx

```diff
@@ -36,35 +37,39 @@ export class DialogRenderer extends Component<Props> {
 
   render() {
     const { theme, closeDialog } = this.props;
-
     return (
       <Fragment>
         {this.props.dialogs.map((dialog) => (
```

Comment:
is this happy with the types? does it know that `tagName` exists below? sometimes it has problems with `map()`

Reply:
also you can add `dialogs` above in line 38 and then refer to it here without `this.props` - looks like that was something that already existed, and maybe there's a reason (shadowing?) to avoid `dialogs` but if not you can it

Reply:
Types seem to be fine here. 
I added `dialogs` as you mentioned and that works fine, thank you.